vcell-cli/src/main/java/org/vcell/cli/run/OmexHandler.java

```diff
@@ -24,18 +24,24 @@ public class OmexHandler {
     CombineArchive archive;
 
     // Assuming omexPath will always be absolute path
+    // NB: Need to convert class to use Log4j2
     public OmexHandler(String omexPath, String outDir) throws IOException {
         try {
-            ResourceUtil.setNativeLibraryDirectory();
-            NativeLib.combinej.load();
-//            System.load("combinej");
+            try {
+                ResourceUtil.setNativeLibraryDirectory();
+                NativeLib.combinej.load();
+            } catch (Exception e){
+                System.err.println("Unable to link to native 'libCombine' lib, check native lib. Attemping alternate solution...");
+                NativeLib.combinej.directLoad();
+            }
         } catch (UnsatisfiedLinkError ex) {
             System.err.println("Unable to link to native 'libCombine' lib, check native lib: " + ex.getMessage());
-            System.exit(1);
-        } catch (Exception e) {
-            System.err.println("Error occurred while importing libCombine: " + e.getMessage());
-            System.exit(1);
+            throw ex;
+        } catch (Exception ex) {
+            System.err.println("Error occurred while importing libCombine: " + ex.getMessage());
+            throw ex;
```

Comment:
looks like OmexHandler constructor declares throwing IOException, but this throws Exception as well (how does this compile?)

Reply:
I'm 95% sure its because the caught exceptions that can make it there are all IOException; So even if we use the more generic Exception, java knows its safe.

I'll change line 40 to be explicitly IOException anyway; cleaner choice.

Reply:
Ahh, I was wrong; There's code lower down that explicitly need IOException; any exception caught by the line 40 Exception statement will be RuntimeExceptions; I fixed that line accordingly